src/main/java/domen/rideapp/domain/model/Ride.java

```diff
@@ -27,6 +30,23 @@ public Ride(String customer, Localization localization, RideStatus status, Price
         this.price = price;
     }
 
+    @JsonCreator
+    public Ride(
+            @JsonProperty("id") String id,
```

Comment:
Postaraj się oduzależnić "domain" od jacksona, w tym przypadku możesz użyć [jackson mixin](https://medium.com/@shankar.ganesh.1234/jackson-mixin-a-simple-guide-to-a-powerful-feature-d984341dc9e2)